tldw_Server_API/app/core/AuthNZ/repos/org_stt_settings_repo.py

```
@@ -0,0 +1,230 @@
+
+    async def ensure_tables(self) -> None:
+        """Ensure the org STT settings table exists when a pool is available."""
+        try:
+            if self._is_postgres():
+                from tldw_Server_API.app.core.AuthNZ.pg_migrations_extra import (
+                    ensure_org_stt_settings_pg,
+                )
+
+                ok = await ensure_org_stt_settings_pg(self.db)
+                if not ok:
+                    raise RuntimeError("PostgreSQL org_stt_settings schema ensure failed")
```

Ruff (0.15.9)

[warning] Abstract `raise` to an inner function

(TRY301)